model/players.js

```diff
@@ -117,9 +119,6 @@ module.exports = {
   all: getAll,
   //TODO: Should this involve a callback?
   get: getPlayer,
-  // TODO: getName is definitely kind of a hack, and illustrates how bad the old data system is.
-  // TODO: Finish refactor of getName -> all-names.nameForKey
-  getName: (key) => nameLookup[key] || 'MISSING KEY',
```

Comment:
Verified that there are no more instances of `getName` in the code base, making the refactor easy to complete.